status/apps/v1/longrunning.go

```diff
@@ -1,17 +1,113 @@
 package v1
 
 import (
+	"fmt"
+
+	"github.com/caicloud/clientset/listerfactory"
 	releaseapi "github.com/caicloud/clientset/pkg/apis/release/v1alpha1"
 	podstatus "github.com/caicloud/clientset/util/status"
-
-	"k8s.io/api/core/v1"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/runtime"
 )
 
-// JudgeLongRunning ...
-//
+type longRunning struct {
+	factory  listerfactory.ListerFactory
+	delegate LongRunning
+	obj      runtime.Object
+	events   []*corev1.Event
+}
+
+func newLongRunning(factory listerfactory.ListerFactory, obj runtime.Object) (*longRunning, error) {
+	var delegate LongRunning
+	var namespace string
+	switch resource := obj.(type) {
+	case *appsv1.Deployment:
+		namespace = resource.Namespace
+		delegate = newDeploymetLongRunning(resource)
+	case *appsv1.DaemonSet:
+		namespace = resource.Namespace
```

Comment:
```suggestion
		namespace = resource.Namespace
```
少了 delegate = newDeploymetLongRunning(resource)